model2vec/distill/tokenizer.py

```diff
@@ -68,7 +68,17 @@ def remove_tokens(tokenizer: Tokenizer, tokens_to_remove: list[str]) -> Tokenize
         tokenizer_data["model"]["vocab"] = reindexed
 
     elif model_type == "Unigram":
-        logger.warning("Removing tokens from a unigram tokenizer is not supported.")
+        # Vocab is a list of tuples.
+        vocab: list[dict[str, Any]] = tokenizer_data["model"]["vocab"]
```

Comment:
The typing here is incorrect, a unigram tokenizer has `list[tuple[str, float]]` as a vocab.